js/router.js

```diff
@@ -1,14 +1,15 @@
 import Vue from 'vue'
 import Router from 'vue-router'
+import {generateUrl} from 'nextcloud-server/dist/router'
```

Comment:
Spaces are missing here :grin:

Reply:
hehe, yeah, the coding style is quite "creative" right now. @kevgathuku has added a linter in #1172 but we're not yet enforcing it. Will try to gradually move files to a uniform style soon :wink: